### PR TITLE
add FN F2000

### PR DESCRIPTION
**Pull request by ghost**
#### Summary

```SUMMARY: [Content] "[Adds new .223 assault rifle FN F2000]"```

#### Testing

Supposed to be a compact weapon, is the volume / weight ok?

### CODE REVIEW

**Comment by LaVeyanFiend** [comment]
Why are there all those other non-F2000 changes? 🤔

File: data/json/items/gun/223.json [hunk 1]
```diff
@@ -45,6 +45,40 @@
     "min_cycle_recoil": 1350,
     "magazines": [ [ "223", [ "stanag30", "stanag50", "survivor223mag" ] ] ]
   },
+  {
+    "id": "fn_f2000",
+    "copy-from": "rifle_auto",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": "FN F2000",
+    "description": "A fully ambidextrous Belgian bullpup rifle chambered in NATO 5.56x45mm rounds, this futuristic looking assault rifle features a compact and modular design.",
```
Comment (tenmillimaster) [inline data/json/items/gun/223.json]: Not calling it the boat? shame.

**Comment by tenmillimaster** [comment]
This should be the semi-auto FS2000 version, not the FN2000.

**Comment by tenmillimaster** [comment]
Moved to #36436

**Pull request closed without merging**